scripts/build.js

```diff
@@ -11,18 +11,19 @@ import getPort, { portNumbers } from 'get-port';
 import ora from 'ora';
 import util from 'util';
 
-const { bundle, copydir, dir, serve, types } = commandLineArgs([
-  { name: 'bundle', type: Boolean },
+const { copydir, serve } = commandLineArgs([
   { name: 'copydir', type: String },
   { name: 'serve', type: Boolean },
-  { name: 'types', type: Boolean }
 ]);
 const outdir = 'dist';
+const cdndir = 'cdn'
```

Comment:
We need to add "cdndir" to `cspell.json`. You can use `npm spellcheck` to test spelling (it doesn't currently block the build). There's a great extension for VS Code that highlights spelling errors. There's probably a CSpell extension for vim as well.